src/routes/[dictionaryId]/_ExportingButton.svelte

```diff
@@ -0,0 +1,43 @@
+<script lang="ts">
+  import { saveAs } from 'file-saver';
+  import JSZip from 'jszip';
+  import { _ } from 'svelte-i18n';
+
+  export let images: string[];
+
+  function downloadHandler() {
```

Comment:
You'll find this code much simpler to write if you use async-await, especially as you continue to build this out.

Reply:
Ok, I'll change this to async/await

Reply:
You were right! Async/Await is so much easier! I was getting a hard time changing things but I was able to figure out how to do it and is very simple :D